assets/l10n/app_en.arb

```diff
@@ -367,7 +367,7 @@
   },
   "loginAddAnAccountPageTitle": "Add an account",
   "@loginAddAnAccountPageTitle": {
-    "description": "Page title for screen to add a Zulip account."
+    "description": "Title for page to add a Zulip account."
```

Comment:
Is it intentional that we omit "the" before "page" here?

Reply:
It's intentional in the sense that lots of other descriptions are in this headline-ese anarthrous (i.e. without articles) style, and I didn't feel this was the time to try to unify everything in that respect :slightly_smiling_face: